code/modules/antagonists/hivemind/hivemind.dm

```diff
@@ -162,31 +154,9 @@
 /datum/antagonist/hivemind/roundend_report()
 	var/list/result = list()
 
-	var/greentext = TRUE
-
 	result += printplayer(owner)
 	result += "<b>Hive Size:</b> [hive_size]"
-
-	var/objectives_text = ""
-	if(objectives.len)
-		var/count = 1
-		for(var/datum/objective/objective in objectives)
-			if(objective.check_completion())
-				objectives_text += "<br><B>Objective #[count]</B>: [objective.explanation_text] <span class='greentext'>Success!</span>"
-			else
-				objectives_text += "<br><B>Objective #[count]</B>: [objective.explanation_text] <span class='redtext'>Fail.</span>"
-				greentext = FALSE
-			count++
-
-	result += objectives_text
-
-	var/special_role_text = lowertext(name)
-
-	if(greentext)
-		result += "<span class='greentext'>The [special_role_text] was successful!</span>"
-	else
-		result += "<span class='redtext'>The [special_role_text] has failed!</span>"
-		SEND_SOUND(owner.current, 'sound/ambience/ambifailure.ogg')
+	result += printobjectives(objectives)
```

Comment:
Print objectives doesn't print the overall win/lose, assuming you still want it.